### ISSUE TITLE
IPhone 6 just jail broke cydia just crashes

### ISSUE THREAD

**Issue by barkus23** [kpwn/yalu102#399]
just jail broke my phone and cydia hasn't worked it just crashes I've tried to reboot it and done the app again but still hasn't worked i can't really code so please give me other suggestions :)

**Comment by ghost** [comment]
same with iphone 6s plus. 

no icloud
no pass
phone was in airplane mode....

typed email and pass correct. jb succesfull...

cydia crashes.

restarted phone. yalu doesnt open... so i deleted it and jb again. yalu doesnt open again..

this time. settings phone app and home bar went.. restoring phone with saved blobs..

**Comment by barkus23** [comment]
Did it work

**Comment by ghost** [comment]
i managed to restore to ios 10.2 because i saved blobs before.. but i am not going to use yalu ever again... it is so buggy...

**Comment by XAVERRenogadeAngle** [comment]
Don't nobody is telling you to. Don't have a jailbreak

**Comment by ghost** [comment]
yeah actually some of you people said that it is fucking sooo stable that i can trust and use.

**Comment by timonline** [comment]
@barkus23 and @LastPust if still interested in solving Cydia crash contact me on twitter @timourrashed.

I know the solution and helped like 10 people so far

**Comment by ghost** [comment]
Don't complain about jailbreak stability here. That is Cydia's fault, not the jailbreak. The jailbreak is as stable as it can get, as said by Luca himself.

#162 #64

**Comment by ghost** [comment]
no it is not

**Comment by barkus23** [comment]
Does this work on iOS 10.2.1 or is there a jailbreak for it

**Comment by ghost** [comment]
@LastPust https://twitter.com/qwertyoruiopz/status/826113297657241600

**Comment by parad0xnasha** [comment]
I'm having the same issue - Cydia just crashes.
Have reinstalled Yalu numerous times, no avail
iPhone 6 iOS 10.1.1

**Comment by ghost** [comment]
#64

**Comment by FloatingGuy** [comment]
cydia crashes maybe because jailbreak failed，i try three times，i found that if  my phone don't auto  reboot after jailbreak ,cydia can work normal.  but  it's difficult not to reboot

**Comment by stevezaluk** [comment]
@barkus23 read the compatibility list bud.

**Comment by tangalbert919** [comment]
Speaking of Cydia, this ipa file is running an outdated version of said application (this one has 1.1.27, and the latest is 1.1.30). I'll see if I can update it.
By the way, there is no 32-bit Cydia after 1.1.27, because since 1.1.28, Cydia is compiled in 64-bit. Bye-bye, 32-bit people! :cry:

**Comment by tangalbert919** [comment]
@zbduid12  this is 2017. People are SUPER lazy to read.
@barkus23 there is not a jailbreak for iOS 10.2.1 yet. I say "yet" because somebody might make one, but it probably won't be Luca making it this time.
@LastPust this jailbreak is as stable as it gets. Cydia, on the other hand, still has iPhoneOS 2.0 support, which is really sad, and it's not easy to develop an application that continues to support an obsolete version of Apple's mobile operating system.
Also, this is not a code-related issue. Why this is still open makes me wonder if anyone even wants to contribute to this anymore.
